### PR TITLE
Updated caption to be more descriptive

### PR DESCRIPTION
**Pull request by aaronjae**
Changed caption to 'Ninja Warrior'

Resolves
Fixes
Closes #2

### CODE REVIEW

File: _slides/03-aaronjae.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Ninja Warrior
```
Comment (JeffreyLew) [inline _slides/03-aaronjae.md:6]: Add "American"  to make it "American Ninja Warrior".

Comment (aaronjae) [reply _slides/03-aaronjae.md:6]: He's Korean!

Comment (JeffreyLew) [reply _slides/03-aaronjae.md:6]: Okay, be specific then.  "Korean Ninja Warrior".

Comment (jeffjohnsonwedbush) [inline _slides/03-aaronjae.md:6]: test caption
```suggestion
Korean American Ninja Warrior
```

**Pull request merged**